contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -533,7 +526,7 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     function getFeeInfo(
         bytes32 destinationChainID,
         uint256 messageID
-    ) external view returns (address feeAsset, uint256 feeAmount) {
+    ) external view returns (address, uint256) {
```

Comment:
Either leave these in or describe the return values in the comment above.

Reply:
Just went through and double checked that we've removed all named returned parameters to stay consistent, as recommend in the audit report. I also updated some of the "@return" comments in the interface definition. To avoid duplication I'll leave the comments here that refer to the documentation in the interface as is 👍

Reply:
Ahhh okay. Didn't realize they were in the interface. Thanks!

Reply:
They we're complete as is, so thanks for calling it out 👍